spec-tests/src/test/java/org/candlepin/spec/SystemAdminSpecTest.java

```diff
@@ -0,0 +1,358 @@
+            EntitlementDTO.class);
+
+        assertThat(userClient.entitlements().getEntitlement(ent1.getId()))
+            .isNotNull()
+            .returns(ent1.getId(), EntitlementDTO::getId);
+        assertThat(userClient.consumers().listEntitlements(consumer1.getUuid())).hasSize(1);
+
+        assertThat(userClient.entitlements().getEntitlement(ent2.getId()))
+            .isNotNull()
+            .returns(ent2.getId(), EntitlementDTO::getId);
+        assertThat(userClient.consumers().listEntitlements(consumer2.getUuid())).hasSize(1);
+    }
+
+    @Test
+    public void shouldOnlyUnsubscribeTheirSystemsEntitlementsUser2() {
+        OwnerDTO owner = ownerApi.createOwner(Owners.random());
+        UserDTO user = createUserType2(owner);
+        ApiClient userClient = ApiClients.basic(user);
+
+        ConsumerDTO consumer1 = userClient.consumers().createConsumer(
+            Consumers.random(owner).username(user.getUsername()));
+        ConsumerDTO consumer2 = client.consumers().createConsumer(
+            Consumers.random(owner).username("admin"));
+
+        PoolDTO pool = createPool(owner);
+        EntitlementDTO ent1 = ApiClient.MAPPER.convertValue(
+            userClient.consumers().bindPool(consumer1.getUuid(), pool.getId(), 1).get(0),
+            EntitlementDTO.class);
+        EntitlementDTO ent2 = ApiClient.MAPPER.convertValue(
+            client.consumers().bindPool(consumer2.getUuid(), pool.getId(), 1).get(0),
+            EntitlementDTO.class);
+
+        userClient.consumers().unbindByEntitlementId(consumer1.getUuid(), ent1.getId());
+        // We can view consumer2 but not remove the entitlement
+        assertForbidden(() -> userClient.consumers().unbindByEntitlementId(
+            consumer2.getUuid(), ent2.getId()));
+    }
+
+    private UserDTO createUserType1(OwnerDTO owner) {
+        return UserUtil.createWith(client,
+            new PermissionBlueprintDTO().type("USERNAME_CONSUMERS")
+                .owner(new NestedOwnerDTO().id(owner.getId()).key(owner.getKey())),
+            new PermissionBlueprintDTO().type("OWNER_POOLS")
+                .owner(new NestedOwnerDTO().id(owner.getId()).key(owner.getKey())),
+            new PermissionBlueprintDTO().type("ATTACH")
+                .owner(new NestedOwnerDTO().id(owner.getId()).key(owner.getKey())));
+    }
+
+    private UserDTO createUserType2(OwnerDTO owner) {
+        return UserUtil.createWith(client,
+            new PermissionBlueprintDTO().type("USERNAME_CONSUMERS").access("ALL")
+                .owner(new NestedOwnerDTO().id(owner.getId()).key(owner.getKey())),
+            new PermissionBlueprintDTO().type("OWNER").access("READ_ONLY")
+                .owner(new NestedOwnerDTO().id(owner.getId()).key(owner.getKey())),
+            new PermissionBlueprintDTO().type("ATTACH")
+                .owner(new NestedOwnerDTO().id(owner.getId()).key(owner.getKey())));
```

Comment:
You can use Permissions utility to simplify creation of permissions. E.G. This could be written as
```java
return UserUtil.createWith(client,
    Permissions.USERNAME_CONSUMERS.all(owner),
    Permissions.OWNER.readOnly(owner),
    Permissions.ATTACH.all(owner));
```